main.py

```diff
@@ -2,6 +2,35 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    for genre in ("Western", "Action", "Dramma"):
+        Genre.objects.create(name=genre)
+
+    for first_name, last_name in (
+            ("George", "Klooney"),
+            ("Kianu", "Reaves"),
+            ("Scarlett", "Keegan"),
+            ("Will", "Smith"),
+            ("Jaden", "Smith"),
+            ("Scarlett", "Johansson")
+    ):
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Dramma", ).update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu").update(
+        first_name="Keanu", last_name="Reeves")
+
```

Comment:
It would be more readable if you will try to do something like: 
```suggestion
    Actor.objects.filter(first_name="Kianu") \
        .update(first_name="Keanu", last_name="Reeves")

```

Same for the return below.